spec/api-web-contents-spec.js

```diff
@@ -116,6 +116,23 @@ describe('webContents module', () => {
     })
   })
 
+  describe('isCurrentlyAudible() API', () => {
+    it('returns whether audio is playing', (done) => {
+      w.loadURL(`file://${path.join(__dirname, 'fixtures', 'api', 'is-currently-audible.html')}`)
+      w.show()
+      w.webContents.once('did-finish-load', () => {
+        assert.equal(w.webContents.isCurrentlyAudible(), false)
+
+        ipcMain.once('playing', () => {
+          assert.equal(w.webContents.isCurrentlyAudible(), true)
+          done()
+        })
+
+        w.webContents.send('play')
+      })
+    })
+  })
```

Comment:
Let's use [our event helpers](https://github.com/electron/electron/blob/master/spec/events-helpers.js) and Chai's [`expect`](http://www.chaijs.com/api/bdd/) here:

```js
const chai = require('chai')
const dirtyChai = require('dirty-chai')

const {expect} = chai

chai.use(dirtyChai)

<...>

it('returns whether audio is playing', async () => {
  w.loadURL(`file://${path.join(__dirname, 'fixtures', 'api', 'is-currently-audible.html')}`)
  w.show()
  await emittedOnce(w.webContents, 'did-finish-load')
 
  expect(w.webContents.isCurrentlyAudible()).to.be.false()

  w.webContents.send('play')
  await emittedOnce(ipcMain, 'playing')

  expect(w.webContents.isCurrentlyAudible()).to.be.true()
})
```